libraries/extractor/src/main/java/androidx/media3/extractor/DtsUtil.java

```diff
@@ -41,6 +41,17 @@ public final class DtsUtil {
   private static final int SYNC_VALUE_14B_BE = 0x1FFFE800;
   private static final int SYNC_VALUE_LE = 0xFE7F0180;
   private static final int SYNC_VALUE_14B_LE = 0xFF1F00E8;
+  /**
+   * DTS Extension Substream Syncword (in different Endianness) is defined in Section 7.4.1 of:
```

Comment:
Could you please reword this to align it with the other comments in this file? For example:

"DTS Extension Substream Syncword (in different endianness). See ETSI TS 103 491 section 7.4.1."

No need to put the whole link I think. The same applies to the other new comment below.

Reply:
@tianyif Done. Please see 9a1bde9e5f4e4fdfa407d8630d555bd563b5c1be. Thanks